src/autocomplete/dataset.js

```diff
@@ -127,7 +138,13 @@ _.mixin(Dataset.prototype, EventEmitter, {
       var suggestionsHtml = html.suggestions.
         replace('%PREFIX%', this.cssClasses.prefix).
         replace('%SUGGESTIONS%', this.cssClasses.suggestions);
-      $suggestions = DOM.element(suggestionsHtml).css(this.css.suggestions);
+      $suggestions = DOM
+        .element(suggestionsHtml)
+        .css(this.css.suggestions)
+        .attr({
+          role: 'group',
+          'aria-label': this.name
```

Comment:
In docsearch this value ends up being `1` and attached to "ds-suggestions". I might have to remove this.